PlaygroundAppUITests/XCUIDevice+UserInterfaceStyleTests.swift

```diff
@@ -0,0 +1,12 @@
+import XCUITestHelpers
+import XCTest
+
+class XCUIDeviceUserInterfaceStyleTests: XCTestCase {
+
+    func testReadingUserInterfaceStyleReturnsConsistentValues() {
+        // We can't control the device appearance mode (light vs dark) from the tests, so we just
+        // check the method returns consistent results
+        XCTAssertNotEqual(XCUIDevice.userInterfaceStyle, .unspecified)
```

Comment:
After reading about [UIUserInterfaceStyle](https://developer.apple.com/documentation/uikit/uiuserinterfacestyle), I'm wondering why the test checks that the value is not equal to `unspecified`, which is similar to `System default`, as I understand, and is one of the possible values?

Is this because of the settings of this particular PlaygroundApp? I failed to run the test because of the `cyclic dependency declaration found: XCUITestHelpersTests -> XCUITestHelpersTests` error showing when I opened this package repo in separate, but not as a part of other workspace, which I couldn't solve.

Reply:
>  I'm wondering why the test checks that the value is not equal to `unspecified`

These tests may run on either a device (iPad or iPhone) or a Simulator (again, iPad or iPhone). The tests themselves cannot, without additional logic, check what kind of device they are running on. But, they should always be running on something that is _not_ `unspecified`.

Does that make more sense? 🤔 

It's a bit redundant given the other expectation but I felt that, since I didn't have a proper test for the computed value, two different assertions might be better than one.

---

As for the cyclic dependency: good catch! I opened https://github.com/Automattic/XCUITestHelpers/pull/2 for it. Thanks!

Reply:
> But since this code is checking what the device uses, the returning value must be either light or dark. Maybe... 😅

Hm, yes, this actually makes a lot of sense: `unspecified` results in UI being either `light` or `dark`, and, as I understand, we are checking for that in the test. Thank you!